model/src/game_mechanics/pets.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+use crate::*;
+pub use crate::game_mechanics::skills;
+pub use crate::authenticated::characters::PetId;
```

Comment:
Feel free to move the definition of `PetId` over to this module, not required though